modules/packer/custom-image/README.md

```diff
@@ -48,16 +48,6 @@ The [Image Builder] blueprint demonstrates a solution that builds an image using
 
 Please review the [examples README] for usage instructions.
 
-### Intel-Optimized Slurm Cluster
-
-The [Intel-Optimized] Slurm Cluster [blueprint](../../../community/examples/intel/hpc-intel-select-slurm.yaml)
-adds the Intel compliance software on top of a Slurm on GCP image.
-
-[Image Builder]: ../../../examples/image-builder.yaml
```

Comment:
@harshthakkar01 I think you may be accidentally breaking some links here. Look for `[Image Builder]` above and it will rely on the target definition on Line 56.

Reply:
Thanks for catching this. I moved this one at the top.